templates/zerver/help/mastering-the-compose-box.md

```diff
@@ -1,24 +1,72 @@
 # Mastering the compose box
 
-When composing a message, Zulip lets you view a different conversation from the
-one you are composing to. For example, you can start a new topic without
+When composing a message, Zulip lets you start a new topic without
 changing your narrow, send a private message about the topic you're viewing, or
 look up a related discussion.
 
-In this context, the parts of the message view that are outside of the
+In this context, you can view a different conversation from the one you are
+composing to. The parts of the message view that are outside of the
 conversation you are composing to are faded for clarity.
 
 ## Go to conversation
 
-To narrow the view to the conversation you're currently composing to,
-click on the **Go to conversation** (<i class="zulip-icon
-   zulip-icon-arrow-left-circle"></i>) button in the compose box.
+Zulip also lets you narrow the view to the conversation you're currently
+composing to.
 
-You can also use the keyboard shortcut <kbd>Ctrl</kbd> + <kbd>.</kbd>
-to go to the conversation to which you are composing.
+{start_tabs}
 
-While the button is only shown when composing to a stream, the keyboard shortcut
-will work in both stream messages and private messages.
+{!start-composing.md!}
+
+1. Click the **Go to conversation** (<i class="zulip-icon
+   zulip-icon-arrow-left-circle"></i>) button to go to the stream that you
```

Comment:
This is incorrect. It doesn't take you to the stream -- it takes you to the *conversation*, which in this case would be a topic.

Reply:
How about:

> If composing a stream message, click the **Go to conversation** (<i class="zulip-icon
   zulip-icon-arrow-left-circle"></i>) button in the compose box.

We've already explained above what going to a conversation means, and the button location hint seems helpful.